### PR TITLE
Convert thrown exception messages to plain text in HTML and Form packages

### PR DESCRIPTION
**Pull request by eddieajau**
Removed the use of JText in hard thrown exception messages that are runtime errors (something the developer needs to fix).

### CODE REVIEW

File: libraries/joomla/html/html.php [hunk 1]
```diff
@@ -138,7 +138,7 @@ public static function _($key)
 		}
 		else
 		{
-			throw new InvalidArgumentException(JText::sprintf('JLIB_HTML_ERROR_NOTSUPPORTED', $className, $func), 500);
+			throw new InvalidArgumentException(JText::sprintf('%s::%s not found.', $className, $func), 500);
```
Comment (elkuku) [inline libraries/joomla/html/html.php]: I think you missed a JText here ;)

**Comment by AmyStephen** [comment]
@eddieajau - good to see this, it answers one of my questions about the separation. If you want help doing this in other packages, I am happy to do so.

**Comment by eddieajau** [comment]
Ooops. Thanks. Fixed.

@AmyStephen that would be great if you can go through the other thrown exceptions.

**Comment by AmyStephen** [comment]
@eddieajau Great - will do so as soon as my fork spot is available when this pull request is either processed or closed. https://github.com/joomla/joomla-platform/pull/970

**Pull request merged**